db/migration/V4.14.0.1__5822-export-var-meta.sql deployment failures on develop

`[2019-05-20T18:56:39.496+0000] [glassfish 4.1] [SEVERE] [] [javax.enterprise.system.tools.deployment.common] [tid: _ThreadID=46 _ThreadName=admin-listener(5)] [timeMillis: 1558378599496] [levelValue: 1000] [[
  Exception while invoking class org.glassfish.ejb.startup.EjbApplication start method
javax.ejb.EJBException: javax.ejb.CreateException: Initialization failed for Singleton StartupFlywayMigrator`

`Caused by: org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: 
Migration V4.14.0.1__5822-export-var-meta.sql failed
SQL State  : 42701
Error Code : 0
Message    : ERROR: column "postquestion" of relation "variablemetadata" already exists
Location   : db/migration/V4.14.0.1__5822-export-var-meta.sql`

`Caused by: org.flywaydb.core.internal.sqlscript.FlywaySqlScriptException: 
Migration V4.14.0.1__5822-export-var-meta.sql failed
SQL State  : 42701
Error Code : 0
Message    : ERROR: column "postquestion" of relation "variablemetadata" already exists
Location   : db/migration/V4.14.0.1__5822-export-var-meta.sql`

etc. observed via EC2-create/dataverse-ansible/phoenix

--- a/src/main/resources/db/migration/V4.14.0.1__5822-export-var-meta.sql
+++ b/src/main/resources/db/migration/V4.14.0.1__5822-export-var-meta.sql
@@ -1,2 +1,2 @@
 ALTER TABLE variablemetadata
-ADD COLUMN postquestion text;
+ADD COLUMN IF NOT EXISTS postquestion text;

